### PR TITLE
feat(plasma-icons): Add icons for `36 dp` from G to O

### PR DESCRIPTION
**Pull request by neretin-trike**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.64.0-canary.11.2305355661.0
  npm install @salutejs/plasma-core@1.58.0-canary.11.2305355661.0
  npm install @salutejs/plasma-icons@1.82.0-canary.11.2305355661.0
  npm install @salutejs/plasma-temple@1.66.0-canary.11.2305355661.0
  npm install @salutejs/plasma-typo@0.5.0-canary.11.2305355661.0
  npm install @salutejs/plasma-ui@1.102.0-canary.11.2305355661.0
  npm install @salutejs/plasma-web@1.99.0-canary.11.2305355661.0
  npm install @salutejs/plasma-cy-utils@0.11.0-canary.11.2305355661.0
  # or 
  yarn add @salutejs/plasma-b2c@1.64.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-core@1.58.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-icons@1.82.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-temple@1.66.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-typo@0.5.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-ui@1.102.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-web@1.99.0-canary.11.2305355661.0
  yarn add @salutejs/plasma-cy-utils@0.11.0-canary.11.2305355661.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-11/

File: packages/plasma-icons/src/scalable/Icon.assets.36/LocationOldOutline.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { IconProps } from '../IconRoot';
+
+export const LocationOldOutline: React.FC<IconProps> = (props) => (
+    <svg width="100%" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+        <path
+            opacity="0.95"
```
Comment (Yeti-or) [inline packages/plasma-icons/src/scalable/Icon.assets.36/LocationOldOutline.tsx:8]: убрать бы

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-11/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-11/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-11/
  ui storybook: http://plasma.sberdevices.ru/pr-11/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-11/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-11/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-11/temple-storybook/

**Pull request merged**